src/Sarif.FunctionalTests/Sarif.FunctionalTests.csproj

```diff
@@ -16,6 +16,12 @@
     <AssemblyOriginatorKeyFile>..\TestKey.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., build.props))\build.props" />
+  <PropertyGroup>
+    <!-- Override the TargetFrameworkVersion specified in build.props. This project must target
+         .NET 4.6 because the Sarif.Validator project, upon which this project depends, targets
+         that version. -->
+    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
```

Comment:
As we discussed: the VS schema validation library is build against 4.6. We will reconsider this dependency.